### ISSUE TITLE
Subscribe remote public timelines of activitypub instances

### ISSUE THREAD

**Issue by kr428** [friendica/friendica#12890]
Currently, Fedilab (on the client side) offers a feature to subscribe to public timelines of Mastodon / activitypub instances. To remove this from the client, it would be really nice to have this baked directly into Friendica. You think this is possible...?

**Comment by annando** [comment]
I'm not sure how this is done in Fedilab. Possibly they are simply polling the API endpoint for that. I have got the feeling as if this could be really demanding for both the polling and the polled system.

**Comment by kr428** [comment]
Yes, I think this is more or less what happens. Not sure if there's any way to do this in a somewhat conservative way without putting too much load onto both ends of the connection, but it surely would be a neat feature to have. Is it, too, too much different from, say, subscribing to a remote instances RSS feed in Friendica?

**Comment by annando** [comment]
It is some kind of feed (just another format). But a regular feed can be polled every hour, possibly every 10 minutes, to really be up to date. But a public timeline can be really intense. You can try it out by yourself. Have a a look at `https://mastodon.social/api/v1/timelines/public?local=true` (which is the public timeline of mastodon.social). Then after a few seconds reload it and have a look at the changes. You can do this polling on smaller instances where people only are active every few minutes. But on systems with several new posts per second this cannot work.

**Comment by kr428** [comment]
Hmmm... ok, agreed. It seemed a valid idea but actually, indeed, in the (fedilab) frontend the app always will download just a preconfigured batch of updates off there automatically, leaving the user to manually decide to want more (and fetching more if desired). That kind of interaction won't work on the server in a meaningful way I guess... .

**Comment by kr428** [comment]
Closing this. Maybe being able to pull data for new contacts initially would suffice here.
